src/bindings/c/docs/How_to_Debug_C_API_Issues.md

```diff
@@ -0,0 +1,42 @@
+# How to Debug C API Issues
+
+C API provides exception handling, here is all possible return values of the functions:
+```ruby
```

Comment:
ruby?

Reply:
> ruby?

misunderstanding will delete. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting